tests/test_components.py

```diff
@@ -67,6 +68,10 @@ def transpose_state(state, dims=None):
 def call_with_timestep_if_needed(
         component, state, timestep=timedelta(seconds=10.)):
 
+    if isinstance(component, IceSheet):
+        output, diag = component(state, timestep=timestep)
```

Comment:
It might be better to expand "diag" into the full word "diagnostics". It's clearer for new users, and only shows up three times in the code (for this particular instance of `diag`).

Reply:
sounds good!